### PR TITLE
Add description in Thai (th)

### PR DESCRIPTION
**Pull request by melvinmcrn**
@Roshy10 just separated TH into this PR.

### CODE REVIEW

File: src/locales/th.json
Comment (Roshy10) [inline src/locales/th.json:1]: you'll need to add it to /src/i18n.js too for it to be used

**Review by Roshy10 — CHANGES_REQUESTED** [review]
you'll need to add it to /src/i18n.js too for it to be used

**Comment by melvinmcrn** [comment]
Please review again. Thanks. @Roshy10

**Comment by Roshy10** [comment]
you seem to have added a selector in the footer, by default the language set in the browser will be used, so I don't think it's necessary to do that. It will also look a bit messy which I'm not keen on

Please also squash your commits

**Review by Roshy10 — CHANGES_REQUESTED** [review]
see comment above

**Comment by melvinmcrn** [comment]
@Roshy10 all done. please review again.

**Review by Roshy10 — CHANGES_REQUESTED** [review]
make sure to revert the package json and package lock too

**Pull request closed without merging**